packages/transition-common/src/services/service/ServiceDuplicator.ts

```diff
@@ -17,6 +17,7 @@ export interface DuplicateServiceOptions {
     serviceCollection?: ServiceCollection;
 }
 
+// @deprecated Service duplication should be done in the backend, but this function is still used in the frontend by other object's duplication
```

Comment:
We should probably add a Logline somewhere saying we are using a deprecated function

Reply:
It will be removed shortly anyway

Reply:
"shortly" ;)